mm2src/coins/rpc_command/tendermint/staking.rs

```diff
@@ -0,0 +1,149 @@
+use common::{HttpStatusCode, PagingOptions, StatusCode};
+use cosmrs::staking::{Commission, Description, Validator};
+use mm2_core::mm_ctx::MmArc;
+use mm2_err_handle::prelude::MmError;
+
+use crate::{lp_coinfind_or_err, tendermint::TendermintCoinRpcError, MmCoinEnum};
+
+/// Represents current status of the validator.
+#[derive(Default, Deserialize)]
+pub(crate) enum ValidatorStatus {
+    All,
+    /// Validator is in the active set and participates in consensus.
+    #[default]
+    Bonded,
+    /// Validator is not in the active set and does not participate in consensus.
+    /// Accordingly, they do not receive rewards and cannot be slashed.
+    /// It is possible to delegate tokens to a validator in this state.
+    Unbonded,
+}
+
+impl ToString for ValidatorStatus {
+    fn to_string(&self) -> String {
+        match self {
+            ValidatorStatus::All => String::default(),
+            ValidatorStatus::Bonded => "BOND_STATUS_BONDED".into(),
+            ValidatorStatus::Unbonded => "BOND_STATUS_UNBONDED".into(),
+        }
+    }
+}
+
+#[derive(Deserialize)]
+pub struct ValidatorsRPC {
+    #[serde(rename = "ticker")]
+    coin: String,
+    #[serde(flatten)]
+    paging: PagingOptions,
```

Comment:
shouldn't we have paging optional?

Reply:
It's already optional.

Reply:
i meant having it optional as in the sense of accepting it being `None`, and not applying a default.
so the pagination in this request:
https://github.com/KomodoPlatform/komodo-defi-framework/blob/61b165442a2b56469b04db8dda9534148a5cf34c/mm2src/coins/tendermint/tendermint_coin.rs#L2101-L2107
could follow accordingly.

it's OK though if u did this intentionally because we want the default behavior to paginate like so.

Reply:
I think we should not rely on 3rd party defaults for pagination :)